cpp/test/Common/TestHelper.cpp

```diff
@@ -200,7 +200,7 @@ Test::TestHelper::getTestEndpoint(const Ice::PropertiesPtr& properties, int num,
     std::string protocol = prot;
     if (protocol.empty())
     {
-        protocol = properties->getPropertyWithDefault("Ice.Default.Protocol", "default");
+        protocol = properties->getIceProperty("Ice.Default.Protocol");
```

Comment:
This returns `tcp` when Ice.Default.Protocol is not set. The test scripts do set Ice.Default.Protocol.